src/core/prompts/sections/markdown-formatting.ts

```diff
@@ -0,0 +1,32 @@
+export function markdownFormattingSection(): string {
+	return `====
+
+MARKDOWN FORMATTING
+
+When providing responses, use proper markdown formatting:
+
+- Use code blocks with language hints for code snippets:
+  \`\`\`typescript
+  const example = "code here";
+  \`\`\`
+
+- Use bullet points for lists
+- Use headings with appropriate levels (# H1, ## H2, etc.)
+- Format inline code with backticks and clickable links with line numbers
+- Use tables when comparing or listing structured data
```

Comment:
Do we support rendering markdown tables? Feel like I've heard people complain about that

Reply:
Yeah... doesn't render as a table

Reply:
Looks like https://github.com/RooVetGit/Roo-Code/pull/2632 fixes a lot of these things

Reply:
Thank you, I will pull the note about tables and let the guys in #2632 handle that.

Reply:
> Looks like https://github.com/RooVetGit/Roo-Code/pull/2632 fixes a lot of these things

The PR #2632 focuses on markdown rendering improvements for styling and layout. There are no specific changes to _markdown link_ functionality.

Whereas, this PR #3087 focuses _exclusively_ on making it possible to click on a markdown link that takes you to an exact line number within the source code for code-referenced definitions. This PR is intended to be super thin, no major changes, there is no overlap with any other PRs that I am aware of.

Reply:
Sounds good! Will look closely tonight.

Reply:
> Do we support rendering markdown tables? 

update this PR to omit discussion of tables:

> [feat-markdown-open-file-on-click 94f01767] fix: remove markdown table formatting instruction
> Tables are not currently supported. This functionality will be addressed in #2632. Removing the instruction until table support is implemented.